iris-mpc-gpu/tests/e2e.rs

```diff
@@ -571,7 +606,7 @@ mod e2e_test {
         /// The rng that is used internally
         rng:                    StdRng,
 
-        // info for current batch, will be cleared at the start of a new batch
+        /// info for current batch, will be cleared at the start of a new batch
```

Comment:
that was actually intended to be a non-doc comment, being a deliminator between persistent and per-batch storage